buttervolume/plugin.py

```diff
@@ -109,6 +109,44 @@ def volume_list():
                        'Err': ''})
 
 
+@route('/VolumeDriver.Volume.Sync', ['POST'])
+def volume_sync():
+    """Rsync between two nodes"""
+    test = jsonloads(request.body.read()).get('Test', False)
+    remote_volumes = VOLUMES_PATH if not test else TEST_REMOTE_PATH
+    volumes = jsonloads(request.body.read())['Volumes']
+    remote_hosts = jsonloads(request.body.read())['Hosts']
+    port = os.getenv("SSH_PORT", '1122')
+    errors = list()
+    for volume_name in volumes:
+        local_volume_path = join(VOLUMES_PATH, volume_name)
+        remote_volume_path = join(remote_volumes, volume_name)
+        for remote_host in remote_hosts:
+            log.debug(
+                "Rsync volume: %s from host: %s",
```

Comment:
I've chosen to pull data to avoid to manage btrfs subvolume creation on the remote machine